templates/zerver/right_sidebar.html

```diff
@@ -40,7 +40,12 @@
                 <div id="userlist-header">
                   <h4 class='sidebar-title' id='userlist-title'>{{ _('USERS') }}</h4>
                 </div>
-                 <input class="user-list-filter" type="text" placeholder="{{ _('Search people') }}" />
+                <div class="input-append">
+                  <input class="user-list-filter" type="text" placeholder="{{ _('Search people') }}" />
+                  <button type="button" class="btn search_people_button" disabled="disabled">
```

Comment:
If the purpose of this button is to clear the search, then I think the class should be `clear_search_people_button`.  And since there is only one of these in the app, it may make sense to use an id vs. a class.